breeze-complete

```diff
@@ -59,6 +59,7 @@ _BREEZE_ALLOWED_INSTALL_AIRFLOW_VERSIONS=$(cat <<-EOF
 1.10.2
 master
```

Comment:
Actually this is mistake - neither master, nor v1-10-test will work here. INSTALL_AIRFLOW_VERSION installs Airflow from PyPI and you can only find the "numeric" version there. So rather than adding v2-0-test, we should remove those two branches from this liest